src/jit/codegenxarch.cpp

```diff
@@ -4884,9 +4882,8 @@ void CodeGen::genCallInstruction(GenTreePtr node)
                 stackArgBytes += argBytes;
             }
             else
-            {
 #endif // FEATURE_PUT_STRUCT_ARG_STK
-
+            {
                 stackArgBytes += genTypeSize(genActualType(arg->TypeGet()));
```

Comment:
Just a simple fix for if ~ else code block that is surrounded with `#if` directive.  Bigger block is surrounded with ` _TARGET_X86_ || FEATURE_UNIX_AMD64_STRUCT_PASSING` so no effect other arch/defines.